docker/etc/mysql/mysqld.cnf

```diff
@@ -1,7 +1,8 @@
 [mysqld]
-character-set-server=utf8
-collation-server=utf8_unicode_ci
+character-set-server=utf8mb4
+collation-server=utf8mb4_unicode_ci
```

Comment:
Shouldn't this be `collation-server=utf8mb4_0900_ai_ci`?

Reply:
Good point @djjuhasz!

I didn't want to change the current collation that much but it may be a good idea to use the latest Unicode version and the case and accent insensitivity.

Extra reading:
https://dev.mysql.com/doc/refman/8.0/en/charset-unicode-sets.html
https://mysqlserverteam.com/new-collations-in-mysql-8-0-0/
https://stackoverflow.com/questions/54885178/whats-the-difference-between-utf8-unicode-ci-and-utf8mb4-0900-ai-ci